Updated index.html; added markup for work and about_myself sections, …

GiftedNathan opened the pull request:
Updated index.html; added markup for work and about_myself sections, 
Updated style.css; added styling for work and about_myself sections,
Updated images folder; added images for work sections and icons for social media.

GiftedNathan committed:
Added about.css; styling for about section

GiftedNathan committed:
Updated index.html; added markup for work and about_myself sections, Updated style.css; added styling for work and about_myself sections

GiftedNathan commented:
Though the design is for mobile version, it responsible enough to adapt to desktop version. For proper review, kindly resize screen size to mobile version.

amejid commented on index.html:
```diff
@@ -4,39 +4,265 @@
+
+            <!-- tonic section  -->
+            <article class="work-card">
+
+                <div class="card-image-mobile-container"> <!-- this will only display on mobile view  -->
+                    <img src="./images/tonic-image-mobile.png" alt="" class="card-image-mobile">
+                </div>
+                
+                <div class="card-content">
+                    <h2>tonic</h2>
+
+                    <div class="name">
+                        <p>canopy</p>
+                        <ul>
+                            <li>Back end dev</li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                    <p class="content-text">
+                        A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required.
+                    </p>
+
+                    <ul class="buttons">
+                        <li><button>html</button></li>
+                        <li><button>css</button></li>
+                        <li><button>javascript</button></li>
+                    </ul>
+
+                    <button class="cta-btn">
+                        see project
+                    </button>
+
+                </div>
+
+                <div class="card-image"> <!-- this will only display on desktop view  -->
+                    <img src="./images/Snapshoot Portfolio.png" alt="" class="card-image-desktop">
+                </div>
+
+            </article>
```
- [ ] Kindly update your styles to match the Figma template. I think your `font-size` and `padding` are a bit too much when compared to the template. Try to update them accordingly. 🤝️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/205427983-c9e33cd4-d099-4482-a782-622f98883f9b.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/205427920-fa977ed2-0733-475b-9afd-0b70f7900069.png" width="300">|

amejid commented on index.html:
```diff
@@ -4,39 +4,265 @@
+                </p>
+
+                <div class="connect">
+                    <p>LET’S connect</p>
```
- [ ] Kindly use an appropriate `hx` tag instead of a `p` tag for this element to make it semantically more correct. 😎️

amejid commented on index.html:
```diff
@@ -4,39 +4,265 @@
+            </article>
+
+            <article class="right-content">
+                <h2>Languages</h2>
+                <ul class="languages">
+                    <li>
+                        <img src="./images/js.png" alt="">
+                        <h4>Javascript</h4>
+                    </li>
+                    <li>
+                        <img src="./images/html.png" alt="">
+                        <h4>HTML</h4>
+                    </li>
+                    <li>
+                        <img src="./images/css.png" alt="">
+                        <h4>CSS</h4>
+                    </li>
                 </ul>
-            </div>
+
+                <p class="tools">framworks</p>
+                <p class="tools">skills</p>
+            </article>
```
- [ ] Since this content is a group of lists, kindly use `ul` and `li` tags to structure it.
The final structure could look like this

```
<ul class="right-content">
  <li>
    Languages
    <ul class="languages">
      <li>
        <img src="./images/js.png" alt="">
        <h4>Javascript</h4>
      </li>
      <li>
        <img src="./images/html.png" alt="">
        <h4>HTML</h4>
      </li>
      <li>
        <img src="./images/css.png" alt="">
        <h4>CSS</h4>
      </li>
    </ul>
  </li>
  <li>
    Frameworks
  </li>
  <li>
    Skills
  </li>
</ul>
```

- [ ] Please try to match the colors to the Figma template. 🥸️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/205428484-e41c9fcf-3587-47f4-a099-6aa0e91b9eaf.png" width="200">|<img src="https://user-images.githubusercontent.com/17243325/205428414-5457509f-27da-448d-b59e-e83d8a8dfc4c.png" width="200">|

amejid reviewed (changes requested):
Hi @GiftedNathan, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<img src="https://media2.giphy.com/media/t8YPf3ACKeouLzEIvr/giphy.gif"/>
## Highlights
- Professional README :heavy_check_mark: 
- Descriptive PR :heavy_check_mark: 
- Good use of Github flow :fire: 
- Nice code structure :100: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

GiftedNathan committed:
Updated about.css; updated card font-size to match figma design, updated background color for about_me section list item.

GiftedNathan commented:
More about the recent commit:
Updated card font-size to match figma design, updated background color for about_me section list item.
Used 'ul' and 'li' tags instead of 'p' tag for tools list in about_me section.

frank1738 reviewed (approved):
Hi @GiftedNathan , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media1.giphy.com/media/StL3vbwXU21ONOA3Ue/giphy.gif"/>

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
